src/components/dashboard/components/Dashboard.js

```diff
@@ -23,6 +25,7 @@ const Dashboard = props => {
             <Content className="dashboard-content">
                 <Route path={path} exact strict component={Overview} />
                 <Route path={`${path}/campaigns`} component={Campaigns} />
+                <Route path={`${path}/locations`} component={Locations} />
```

Comment:
the strings should be placed in a constants file

Reply:
just realized the different paths have been declared as constants in `router/constants`. Make use of the constants here